spec/plans/provision.fmf

```diff
@@ -261,3 +265,18 @@ example: |
             role: client
```

Comment:
I am wondering, shouldn't we update the two existing examples (name & role) with the group syntax as well?

Reply:
Eh, lets disregard this for now. I've just tried adding support for this new syntax and it won't be as easy with the current tmt implementation so I am not going to change it just yet. The parsing of the config will be easier once we introduce the concept of SUTs and SUT groups but parsing it now inside provision would be too tedious/hacky

Reply:
Understood and agree. We can update that later. Plus, I think it would be also a valid use case to use the old syntax for cases when sync is not needed. But for those we should choose some better example names.